package.json

```diff
@@ -43,18 +44,46 @@
   "devDependencies": {
     "@ethereumjs/common": "^2.4.0",
     "@lavamoat/allow-scripts": "^2.3.0",
-    "@metamask/eslint-config": "^3.0.0",
-    "babel-eslint": "^10.1.0",
+    "@ledgerhq/hw-app-eth": "^6.32.0",
+    "@metamask/eslint-config": "^8.0.0",
```

Comment:
Oops I forgot — you might have mentioned this in another repo, but is there a reason why we are using such an old version of our ESLint config? The newer releases have some updates that would be nice to make use of.

Reply:
Good catch, I updated the ESLint config packages to v11 and applied changes to the code where needed